### DISCUSSION TITLE
NNSVS paper is now available at arXiv

### DISCUSSION THREAD

**Discussion by r9y9** [r9y9/nnsvs#148] [Announcements]
Paper: https://arxiv.org/abs/2210.15987
Demo: https://r9y9.github.io/projects/nnsvs/

Ref: https://twitter.com/r9y9/status/1586911167414341632?s=20&t=f_M3V5jLs5HF_xsIUtldwQ

**Comment by r9y9** [comment]
Accepted for ICASSP 2023

**Reply by ftshijt** [reply]
Congratulations!

**Reply by r9y9** [reply]
Thank you!
